### PR TITLE
NIAD-136: enable failover between queues - approved but wrong branch name

### PR DESCRIPTION
**Pull request by bartek-sarul**

### CODE REVIEW

File: mhs/common/mhs_common/workflow/tests/test_asynchronous_express.py [hunk 1]
```diff
@@ -100,12 +95,10 @@ def test_construct_workflow_with_only_outbound_params(self):
                                                              routing=self.mock_routing_reliability)
         self.assertIsNotNone(workflow)
 
-    def test_construct_workflow_with_only_inbound_params(self):
-        workflow = async_express.AsynchronousExpressWorkflow(queue_adaptor=mock.sentinel.queue_adaptor,
-                                                             inbound_queue_max_retries=INBOUND_QUEUE_MAX_RETRIES,
-                                                             inbound_queue_retry_delay=INBOUND_QUEUE_RETRY_DELAY)
-        self.assertIsNotNone(workflow)
-        self.assertEqual(INBOUND_QUEUE_RETRY_DELAY_IN_SECONDS, workflow.inbound_queue_retry_delay)
+    # def test_construct_workflow_with_only_inbound_params(self):
```
Comment (mattd-kainos) [inline mhs/common/mhs_common/workflow/tests/test_asynchronous_express.py]: Does this commented out test need updating?

File: mhs/common/mhs_common/workflow/tests/test_asynchronous_forward_reliable.py [hunk 2]
```diff
@@ -564,24 +547,24 @@ async def test_successful_handle_inbound_message(self, audit_log_mock):
...
-        await self.workflow.handle_inbound_message(MESSAGE_ID, CORRELATION_ID, self.mock_work_description, INBOUND_MESSAGE_DATA)
-
-        self.mock_queue_adaptor.send_async.assert_called_with(
-            {'ebXML': EBXML, 'payload': PAYLOAD, 'attachments': ATTACHMENTS},
-            properties={'message-id': MESSAGE_ID, 'correlation-id': CORRELATION_ID})
-        self.assertEqual([mock.call(MessageStatus.INBOUND_RESPONSE_RECEIVED),
-                          mock.call(MessageStatus.INBOUND_RESPONSE_SUCCESSFULLY_PROCESSED)],
-                         self.mock_work_description.set_inbound_status.call_args_list)
-        mock_sleep.assert_called_once_with(INBOUND_QUEUE_RETRY_DELAY_IN_SECONDS)
+    # @mock.patch('asyncio.sleep')
```
Comment (mattd-kainos) [inline mhs/common/mhs_common/workflow/tests/test_asynchronous_forward_reliable.py]: commented out code

**Review by mattd-kainos — CHANGES_REQUESTED** [review]
Just a few sections where code has been commented out that need attention. Good that testing of queue tries has been moved out of the workflows and into the queue adaptor.

File: Jenkinsfile [hunk 3]
```diff
@@ -145,7 +145,8 @@ pipeline {
                                         --env "AWS_SECRET_ACCESS_KEY=test" \
                                         --env "MHS_DYNAMODB_ENDPOINT_URL=http://dynamodb:8000" \
                                         --env "FAKE_SPINE_ADDRESS=http://fakespine" \
-                                        --env "MHS_INBOUND_QUEUE_URL=http://rabbitmq:5672" \
+                                        --env "MHS_INBOUND_QUEUE_BROKERS=amqp://rabbitmq:5672" \
+                                        --env "MHS_INBOUND_QUEUE_NAME=inbound" \
```
Comment (jaroslaw-wojciechowski) [inline Jenkinsfile:149]: we have the same var at line nr 10:
MHS_INBOUND_QUEUE_NAME = "${ENVIRONMENT_ID}-inbound"

Comment (jaroslaw-wojciechowski) [reply Jenkinsfile:149]: ignore this comment

**Review by mattd-kainos — APPROVED** [review]
changes made

**Review by jaroslaw-wojciechowski — APPROVED** [review]

**Comment by bartek-sarul** [comment]
Closing as branch name is missing `feature/`

**Pull request closed without merging**